src/paginate.js

```diff
@@ -4,6 +4,19 @@ var url = require('url');
 
 module.exports = paginate;
 
+// We want to preserve these query string parameters when paginating.
+var queryParamsToKeep = ['per_page', 'embed', 'q'];
+
+function filterQuery(query) {
+  var filtered = {};
+  queryParamsToKeep.forEach(function(param) {
+    if (query[param]) {
```

Comment:
This doesn't handle ?embed= which is a valid option. I suspect we need to check for it being defined rather than not equivalent to false.